addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -506,16 +503,19 @@ def get_user_batches(self):
         return batches
 
     def unassign_user_batches(self):
-        """ Removes pickings from all batches assigned to the user and cancels the batch
+        """ Get batches for user and unassign them
         """
         # Get user batches
-        batches = self.get_user_batches()
+        self.get_user_batches().unassign()
 
+    def unassign(self):
+        """ Removes incomplete pickings and removes responsible user from batches
```

Comment:
Docstring looks good, but would it make sense to have two methods instead, one to unassign another to remove the incomplete pickings, so to improve naming and functionality clarity?

Reply:
Probably not, it's one method - the act of "unassigning" the batch is to follow the logic of the "unassign" function as defined: It's to unassign the user AND the incomplete pickings IF it's ephemeral

Reply:
👍